packages/httpsnippet-client-api/test/__datasets__/jsonObj-multiline/mock.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  req: {
+    url: 'https://httpbin.org/anything',
+    method: 'post',
+    headers: {
+      'content-type': 'application/json',
+    },
+    functionMatcher: (url, opts) => {
+      return opts.body === JSON.stringify({ foo: 'bar' });
```

Comment:
Would figure we could add `body: {foo: 'bar'}` as a normal matcher but `fetch-mock` doesn't support that.